File: bddtests/src/main/kotlin/mocking/tpp/models/Person.kt
package mocking.tpp.models

import javax.xml.bind.annotation.*

@XmlAccessorType(XmlAccessType.FIELD)
data class Person(
        @XmlAttribute var patientId: String = "default patientId",
        @XmlAttribute var dateOfBirth: String = "default dateOfBirth",
        @XmlAttribute var gender: String = "default gender",
        @field:XmlElement(name = "NationalId") var nationalId: NationalId = NationalId(
                "default type",
                "default value"
        ),
        @field:XmlElement(name = "PersonName") var personName: PersonName = PersonName(
                "default name"
        ),
        @field:XmlElement(name = "Address") var address: TppAddress = TppAddress(
        "default address"
)
)

File: xamarinintegrationtests/NHSOnline.HttpMocks/Tpp/Models/Person.cs
using System.Xml.Serialization;

namespace NHSOnline.HttpMocks.Tpp.Models
{
    public sealed class Person
    {
        [XmlAttribute("patientId")]
        public string? PatientId { get; set; }

        [XmlAttribute("dateOfBirth")]
        public string? DateOfBirth { get; set; }

        [XmlAttribute("gender")]
        public string? Gender { get; set; }

        public NationalId? NationalId { get; set; }

        public PersonName? PersonName { get; set; }

        public Address? Address { get; set; }
    }
}